src/main/java/com/example/MG_RestaurantManager20/product/domain/Recipe.java

```diff
@@ -0,0 +1,42 @@
+package com.example.MG_RestaurantManager20.product.domain;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.beans.Transient;
+import java.util.List;
+import java.util.Map;
+
+@Getter
+@Setter
+@Entity
+@NoArgsConstructor
+public class Recipe {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    private String name;
+
+
+    // private List<Product> products;
```

Comment:
Delete this comment if we don't need this

Reply:
oki, deleted~